Doxyfile

```diff
@@ -0,0 +1,2427 @@
+# Doxyfile 1.8.11
```

Comment:
What's our doxygen methodology?

Reply:
Using move_base_flex as an example - they have the CHANGELOG and README w/ each package.